Scripts/Hue Wheel AIDA64.lua

```diff
@@ -0,0 +1,159 @@
+-- ISC License
+--
+-- Copyright © 2017, Simonas Kazlauskas
+-- Copyright © 2018, Pedro Fanha
+--
+-- Permission to use, copy, modify, and/or distribute this software for any
+-- purpose with or without fee is hereby granted, provided that the above1
+-- copyright notice and this permission notice appear in all copies.
+--
+-- THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+-- REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+-- AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+-- INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+-- LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE
+-- OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+-- PERFORMANCE OF THIS SOFTWARE.
+
+-- cpu, gpu, fan or hue
+local mode = 'hue'
```

Comment:
I think it might be beneficial to no one if you printed a message here saying 'AIDA64 is not installed' if mode ~= 'hue', since in that case the script will do nothing.

Reply:
Ya, would be nice to debug, will add.